packages/mobile/src/Themes/ThemeBuilder.ts

```diff
@@ -0,0 +1,52 @@
+import { IThemeShape } from '../Provider/IThemeShape';
+import { DefaultTheme } from 'react-native-paper';
+import { ITheme } from '@naturacosmeticos/natds-styles';
+
+export function ThemeBuilder(theme: ITheme | undefined, defaultTheme?: ITheme) {
```

Comment:
Change to `export function ThemeBuilder(theme: ITheme | undefined, defaultTheme?: ITheme): IThemeShape`